disasterinfosite/settings.py

```diff
@@ -27,12 +27,6 @@
 else:
     # hazardready.org is the current production server. 23.92.25.126 is its numeric address. eldang.eldan.co.uk is our demo/test server
     ALLOWED_HOSTS = ['hazardready.org', '.hazardready.org', '23.92.25.126', 'eldang.eldan.co.uk']
```

Comment:
We should start taking eldang.eldan.co.uk out because it's outdated.  Does ".hazardready.org" function as a wildcard, covering test.hazardready.org ?